src/apps/api/tests/test_queue_api_permissions.py

```diff
@@ -46,12 +56,7 @@ def test_organizer_can_perform_all_operations(self):
         self.client.login(username='test', password='test')
         response = self.client.put(
```

Comment:
Do we maybe need to mock things here, too?

Reply:
Yes you're right because update will trigger it as well, I created another helper function to make API requests while mocking. Let me know if you think this is gross:
```
    def queue_api_request_with_mock(self, http_method, url_name, url_kwargs, data):
        # So we can make API requests without triggering rabbitmq actual queue creation
        with mock.patch('queues.models.rabbit.create_queue') as rabbit_create_queue:
            rabbit_create_queue.return_value = uuid.uuid4()
            response = getattr(self.client, http_method)(
                reverse(url_name, kwargs=url_kwargs),
                data=data,
                content_type='application/json'
            )
            assert rabbit_create_queue.called
            return response
```
(Having issues with `assert rabbit_create_queue.called` here, still figuring that out)

Reply:
Figured that out, and updated everything appropriately. Helper function returns response, and that. In the appropriate tests we assert it was called.